src/cliAction/buildDeploy/handler_deploy.go

```diff
@@ -124,6 +136,29 @@ func getConfigContent(config RunConfig) (*zBusinessZeropsApiProtocol.StringNull,
 		return nil, err
 	}
 
+	zdk := sdk.New(
```

Comment:
what about to extract calling the endpoint to a separate func? this one seems me to be rather long already ...